wpilibjIntegrationTests/build.gradle

```diff
@@ -24,8 +24,11 @@ dependencies {
     compile 'com.googlecode.junit-toolbox:junit-toolbox:2.0'
     compile 'org.apache.ant:ant:1.9.4'
     compile 'org.apache.ant:ant-junit:1.9.4'
+    compile files(wpilibj.sourceSets.test.output.classesDir)
 }
 
+compileJava.dependsOn tasks.getByPath(':wpilibj:testClasses')
```

Comment:
Different project's testClasses, that's part of wpilibj, not wpilibjIntegrationTests. And anyways, the dependency that exists for free is test depending on compile, not the other way around.